plonk-core/Cargo.toml

```diff
@@ -64,6 +64,9 @@ merlin = { version = "3.0", default-features = false }
 num-traits = { version = "0.2.14" }
 rand_core = {version = "0.6", default-features=false, features = ["getrandom"] }
 
+ark-marlin = "0.3.0"
```

Comment:
In regards of `ark-marlin`, we are just getting `FiatShamirRng`. Don't we have an alternative to save us this dep and more complexity?

Reply:
Yes, I think that in the newer version FiatShamirRng is separated from ark-marlin, will make sure to check and fix :)

Reply:
If possible, I would not recommend using ark-marlin as a dependency to use `FiatShamirRng`, because as CPerezz pointed out, there would be significant refactoring around `FiatShamirRng` in arkworks-rs.

Reply:
What do you recommend using until then?